src/AppInstallerCLICore/Core.cpp

```diff
@@ -78,6 +79,13 @@ namespace AppInstaller::CLI
         // The root command is our fallback in the event of very bad or very little input
         std::unique_ptr<Command> command = std::make_unique<RootCommand>();
 
+        // Load the settings file and warn if necessary.
+        auto warnings = Settings::UserSettings::Instance().GetWarnings();
+        for (const auto& warning : warnings)
+        {
+            context.Reporter.Warn() << warning << std::endl;
```

Comment:
>context.Reporter.Warn() << warning << std::endl; [](start = 12, length = 48)

Print only a warning message and only print the syntax error when winget settings